src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -373,6 +373,11 @@ - (BOOL)applicationShouldTerminateAfterLastWindowClosed:(NSApplication *)theAppl
 	return YES;
 }
 
+- (void)windowDidBecomeKey:(id)arg
+{
+        [myContent sendmouse:0];
```

Comment:
I'm not sure what sending `0` means. The old devdraw called a function `sendmouse()` without arguments.

But, this seems to work, and I see `[myContent sendmouse:0]` in other places as well.

Reply:
0 means mouse buttons are all released, so it will update the location and tell the application no mouse buttons are pressed.